### DISCUSSION TITLE
Used Successfully vs 73 hours of video

### DISCUSSION THREAD

**Discussion by Foxenfurter** [Breakthrough/DVR-Scan#85] [General]
My neighbours were burgled recently and I had 3 days of surveillance to go through. I thought that there maybe some software that could help and I found this , it was a bit of a faff to install because I hadn't got python installed and the exe was buried deep in sub-folders, but managed to run a batch vs 50 security files and get the reviewable content down to < 1 hour.

Commands were like this
C:\Users\jonat\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\Scripts\dvr-scan -i "e:\videorips\security\Zr04rn10 Cam02 20220311231346 021112-1.m4v" -t 0.3 -tb 3s -tp 5s -l 1.2s -o e:\videorips\security\checks\4.avi

NB I didn't get shots of the burglars but did pickup the drone that they were using for snooping.

**Comment by Breakthrough** [comment]
Glad to hear you had success using DVR-Scan, thank you for sharing the parameters you used.  Could you also share the framerate and resolution of the video you processed?

Thank you!
